python37Packages.xdis: 3.8.10 -> 4.0.1

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/python3.7-xdis/versions.



meta.description for python37Packages.xdis is: '"Python cross-version byte-code disassembler and marshal routines"'.


meta.homepage for python37Packages.xdis is: '"https://github.com/rocky/python-xdis/"


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS
- /bin/sh passed the binary check.
- 1 of 1 passed binary check by having a zero exit code.
- 0 of 1 passed binary check by having the new version present in output.
- found 4.0.1 with grep in /nix/store/zk3pycdbzavb5g9qzrpdxxjj9ri324f5-python3.7-xdis-4.0.1
- directory tree listing: https://gist.github.com/dd26160cbdec22684f98030264bf25f8
- du listing: https://gist.github.com/f7250dcaceb96fa2a2832021f2f05aba

</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

16 total rebuild path(s)

4 package rebuild(s)

4 x86_64-linux rebuild(s)
4 i686-linux rebuild(s)
4 x86_64-darwin rebuild(s)
4 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
python27Packages.uncompyle6
python27Packages.xdis
python37Packages.uncompyle6
python37Packages.xdis

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/zk3pycdbzavb5g9qzrpdxxjj9ri324f5-python3.7-xdis-4.0.1 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A python37Packages.xdis https://github.com/r-ryantm/nixpkgs/archive/6f88e9a59b15cb813a7ca5dc94024692df60b60e.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/zk3pycdbzavb5g9qzrpdxxjj9ri324f5-python3.7-xdis-4.0.1
ls -la /nix/store/zk3pycdbzavb5g9qzrpdxxjj9ri324f5-python3.7-xdis-4.0.1/bin
```


</details>
<br/>

r-ryantm committed:
python37Packages.xdis: 3.8.10 -> 4.0.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.7-xdis/versions

c0bw3b commented:
@GrahamcOfBorg build python27Packages.uncompyle6 python27Packages.xdis python37Packages.uncompyle6

c0bw3b committed:
pythonPackages.uncompyle6: 3.2.6 -> 3.3.2

Changes:
https://github.com/rocky/python-uncompyle6/blob/release-python-3.3.2/NEWS.md

c0bw3b commented:
@GrahamcOfBorg build python27Packages.uncompyle6 python27Packages.xdis python37Packages.uncompyle6

c0bw3b commented:
:v: unbreaking two more Py pkgs
